web/src/app/modules/views/side/modules/errors-warnings/components/errors-warnings.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component } from '@angular/core';
+import { ValidationResult } from '../../../../../../validation/validation-result';
+import { ValidationService } from '../../../../../forms/modules/validation/services/validation.service';
+import { AdditionalInformationService } from '../../links-actions/services/additional-information.service';
+
+
+@Component({
+    moduleId: module.id.toString(),
+    selector: 'errors-warnings',
+    templateUrl: 'errors-warnings.component.html',
+    styleUrls: ['errors-warnings.component.css']
+})
+export class ErrorsWarings {
+    private _isCollapsed = false;
+    public set isCollapsed(collapsed: boolean) {
+        this._isCollapsed = collapsed;
+    }
+
+    public get isCollapsed() {
+        if (!this.warnings) {
+            return true;
+        }
+        return this._isCollapsed;
+    }
+
+
+    public visible = true;
+    constructor(private validationService: ValidationService,
+                private additionalInformationService: AdditionalInformationService) { }
+
+
+    private _currentWarningStringSet = new Set();
+    private _currentWarnings: ValidationResult[][] = [];
+    private get warnings() {
```

Comment:
`get warnings` needs to be public (since it is used in a template). In addition, there is no return type (should be `ValidationResult[][]`, I think). Furthermore, I do not really understand the code below, in particular, what the two-dimensional array is about. Please add a clarifying comment to the getter.

```suggestion

    /*
    * Comment?
    */
    public get warnings() : ValidationResult[][] {
```